linguistica/lexicon.py

```diff
@@ -18,7 +20,7 @@
     FileNotFoundError = OSError  # no FileNotFoundError in Python 2
 
 
-class Lexicon:
+class Lexicon(LexiconSignature):
```

Comment:
By class inheritance, the `Lexicon` class has all the signature-related attributes and methods (= member variables and functions in C++ parlance) from the new `LexiconSignature` class.